containers/ws/README.md

```diff
@@ -1,99 +1,153 @@
-# Cockpit on Fedora CoreOS or other container hosts
+# Cockpit WS Container
 
-The standard Fedora and Red Hat Enterprise Linux CoreOS images does not contain
-Cockpit packages.
+This container is suitable for scenarios where you can not or wish not to deploy the Cockpit Web Service and browser
+application natively.
```

Comment:
"you cannot or do not want to deploy..". Let's please also drop the "browser application" -- we don't use this term anywhere, and it's not even clear what that should mean. The browser talks to the cockpit web server.

"natively" also sounds a bit vague -- maybe "through distribution packages (rpm/deb)"?